build.sh

```diff
@@ -10,3 +10,4 @@ done
 # Build assets.
 npm install
 npm run production
+npm run lint || true
```

Comment:
This might be better to put in a Git hook instead as a pre-commit script.

Reply:
Git hook är mest clean men funkar såhär på detta projekt imo!